FeedAPIChallenge/Feed API/FeedItemsMapper.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  FeedItemsMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Mher Movsisyan on 08.08.21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+struct FeedItemsMapper {
+	struct Root: Decodable {
+		private let items: [Item]
+
+		var feedItems: [FeedImage] {
+			items.compactMap { item in
+				guard let uuid = UUID(uuidString: item.imageId),
+							let url = URL(string: item.imageURL) else { return nil }
+
+				return FeedImage(id: uuid, description: item.imageDesc, location: item.imageLoc, url: url)
+			}
+		}
+	}
+
+	struct Item: Decodable {
+		let imageId: String
+		let imageURL: String
+		var imageDesc: String?
+		var imageLoc: String?
+
+		enum CodingKeys: String, CodingKey {
+			case imageId = "image_id"
+			case imageURL = "image_url"
+			case imageDesc = "image_desc"
+			case imageLoc = "image_loc"
+		}
+	}
+
+	static let OK_200 = 200
```

Comment:
`OK_200` is an implementation detail of `FeedItemsMapper`, so it should be marked as `private`:
```suggestion
	private static let OK_200 = 200
```

Reply:
Done